Jellyfin.Api/Helpers/StreamingHelpers.cs

```diff
@@ -287,62 +287,49 @@ private static string GetOutputFileExtension(StreamState state, MediaSourceInfo?
             return ext;
         }
 
-        // Try to infer based on the desired video codec
         if (state.IsVideoRequest)
         {
-            var videoCodec = state.Request.VideoCodec;
+            // Try to infer based on the desired video codec
+            var videoCodec = state.Request.VideoCodec.ToLowerInvariant();
```

Comment:
You should normalize this to uppercase as a best practice. And it might still be slower since it allocates a new string.